### PR TITLE
OSPP 2021-结项报告-开发基于Wechaty实现微俱乐部会员&活动管理的微信小程序

### PR DESCRIPTION
**Pull request by mangguo188**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [ ] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [ ] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [ ] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [ ] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [ ] Select a beautiful and meaningful teaser image for your blog
- [ ] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Comment by huan** [comment]
Please link back to your ospp 2021 project issue.

**Comment by lijiarui** [comment]
Please DO the following steps first:

1. Get your mentor's approval first.
2. Link back to your ospp 2021 project issue.

@atorber

**Comment by mangguo188** [comment]
项目地址：https://github.com/wechaty/summer/issues/82

**Review by lijiarui — CHANGES_REQUESTED** [review]
It seems your blog file is committed in the wrong place, it should not be put under the root directory.

**Review by atorber — APPROVED** [review]

**Review by atorber — APPROVED** [review]

**Review by atorber — APPROVED** [review]

**Comment by lijiarui** [comment]
Hey @atorber 

Please re-approve this PR, this PR is committed to the wrong place, it should be put under `jekyll/_posts/`, but it is put under the root directory.

@mangguo188 Please change as request as soon as possible.

File: 2021-09-29-ospp-final-term-club-management.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+> 国内链接：<https://www.bilibili.com/video/BV1uP4y187T9/>
+
+### 项目PPT：
+
+{% include iframe.html src="/assets/2021/09-final-term-club-managment/final-term.pdf" %}
+
+## 联系我们
+
+- 项目链接：<https://github.com/mangguo188/wechaty-club-management>  
+- 联系方式：1017137803@qq.com
```
Comment (atorber) [inline 2021-09-29-ospp-final-term-club-management.md]: @mangguo188 Please  move this file to the correct file path，Please check carefully before submitting，common sense mistakes should not be made.

**Comment by lijiarui** [comment]
Hey @atorber 

When you approve this PR, I will review this PR. So please review this PR as soon as possible and approve this PR first.

**Review by atorber — APPROVED** [review]

**Comment by atorber** [comment]
> Hey @atorber
> 
> When you approve this PR, I will review this PR. So please review this PR as soon as possible and approve this PR first.

done

**Review by lijiarui — APPROVED** [review]
LGTM

**Pull request merged**